### ISSUE TITLE
Due date defaults to midnight

### ISSUE THREAD

**Issue by happnatious1** [stefan-niedermann/nextcloud-deck#1105]
If you select tomorrow on the calendar for the due date of a card and click OK it defaults to midnight so by the time that tomorrow comes it's already past due. It should default to midnight the next day so you have tomorrow to work on it.

**Comment by stefan-niedermann** [comment]
**TL;DR: We stick to what the Deck server app¹ does for the sake of consistency.**

In order to make the change happen, please raise a [feature request in the repository of the Deck server app](https://github.com/nextcloud/deck/issues/new?assignees=&labels=&template=Feature_request.md). The Android app will follow the behavior of the Deck server app here.

My personal opinion about your proposal:

> by the time that tomorrow comes it's already past due

Which is absolutely correct from my point of view. It is a due date. When the set date is reached, it is *due*. Like a deadline, when it is crossed, it is over. So i have to do my work before this date is reached (Given i did not set a time explicitly of course).

>  It should default to midnight the next day so you have tomorrow to work on it.

No, it shouldn't. There is no "midnight the next day", there is just 00:00 the next day, which is ridiculous because it is the **next** day - not matching the just one second ago set day. It can only be 23:59 of the same day.

One idea might be to default to 12:00 (high noon), because a [push notification](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/FAQ.md#how-to-receive-notifications-when-a-due-date-is-reached) at midnight is usually just nagging and not useful.
Though my personal interpretation and favored solution is to keep 00:00 as default time, therefore referring you to the Deck server app. As mentioned, we will follow no matter what our personally preferred solution is :slightly_smiling_face:

PS.: I am watching the repository of the Deck server app, so i will get notified in case you open an issue there and follow it.

---
¹ Little disclaimer: Please note that the Deck server app is *not* developed by us but by the Nextcloud GmbH itself.
